src/decide.js

```diff
@@ -20,13 +20,16 @@ export class Decide {
         const encoded_data = _.base64Encode(json_data)
         this.instance._send_request(
             `${this.instance.get_config('api_host')}/decide/?v=2`,
-            { data: encoded_data },
+            { data: encoded_data, verbose: true },
             { method: 'POST' },
             (response) => this.parseDecideResponse(response)
         )
     }
 
     parseDecideResponse(response) {
+        if (response?.status === 0) {
+            return
+        }
```

Comment:
What about adding logging something here? The console.log statements in this repo are anyway only be visible when `debug:true` is set.

```suggestion
        if (response?.status === 0) {
            console.error('Failed to fetch feature flags from PostHog.')
            return
        }
```